### ISSUE TITLE
Feature request: Toggle "Click on player name to see details" on and off

### ISSUE THREAD

**Issue by Elonka** [bafolts/terraforming-mars#1125]
First, thanks for all the work on this, my group is loving it!

Right now, we can click on a player's name to see their cards/score, and then have to "X" to close out that screen. Would it be possible to make this a toggle, so click once on their name to open another player's info, and then click again on their name to close it? Thanks!

**Comment by chosta** [comment]
Yes, this one has been bugging me for a while so I guees its time has come :)
